### ISSUE TITLE
bug(core): focus applied when it shouldn't on drag & drop

### ISSUE THREAD

**Issue by platon-rov** [SAP/fundamental-ngx#5095]
#### Is this a bug, enhancement, or feature request?

Bug

#### Briefly describe your proposal.

When drag & dropping items focus isn't cleared after drag and one more focus applied after drop.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

Angular 11.2.7, Fundamental-NGX v0.28.0

#### If this is a bug, please provide steps for reproducing it.

1. Open [any](https://sap.github.io/fundamental-ngx/#/core/list) component which supports our (not Material CDK) Drag & Drop.
2. For the drag & drop items set some styles to have a better visual difference (`background: red` is enough).
2. Drag item, note that focus **isn't** removed, but it should (to remove focus styles from the element).
3. Drop item **after** its initial position - you will see that focus also applied for the item at the drag start position, but it **shouldn't**.

#### Is there anything else we should know?

As a reference you can see how the same behavior [handled](https://stackblitz.com/edit/angular-apxiuj?file=src%2Fapp%2Fcdk-drag-drop-sorting-example.ts) in Material CDK. In short - it removes focus after dragging started and doesn't apply focus after drop.

Focus event fired before dragging that's why it's applied and our goal to reset it, as we cannot skip it.
